E-Commerce/E-Commerce/Presentation/Views/Home/ChartView/CartViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  ChartViewController.swift
+//  E-Commerce
+//
+//  Created by Orlando Nicolas Marchioli on 24/01/2024.
+//
+
+import UIKit
+
+class CartViewController: UIViewController {
+    
+    
+    @IBOutlet weak var cartEmptyImage: UIImageView!
+    @IBOutlet weak var cartTableView: UITableView!
+    var cartViewController: CartViewController?
+    @IBOutlet weak var itemTotal: UILabel!
+    var productsToBuy: [ProductToBuy] = []
+    @IBOutlet weak var confirmButton: UIButton!
+    let verticalCellSpacing: CGFloat = 70.0
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let nib = UINib(nibName: "HomeTableViewCell", bundle: nil)
+        cartTableView.register(nib, forCellReuseIdentifier: "cell")
+        
+        cartTableView.dataSource = self
+        cartTableView.delegate = self
+        
+        confirmButton.layer.cornerRadius = 10
+        
+        itemTotal.text = "0"
+        
+        checkIfHasToBeEnabled()
+        
+        productsToBuy = CartManager.shared.cartItems
+        cartEmptyImage.image = UIImage(systemName: "cart.fill.badge.plus")
```

Comment:
You can extract UI related code like this line to a separate function to make it more easy to understand, such as configureUI()